.gitignore

```diff
@@ -12,5 +12,5 @@ target/
 bin/
 atlassian-ide-plugin.xml
 *.sonar-ide.properties
-*/webapp/css/styles.css
+**/webapp/css/styles.css
```

Comment:
This is really just wallpapering over a symptom of projects generating css in the src directory.  We need a ticket to fix that so this .gitignore line can finally be removed.